authorization-common-api/src/main/java/com/sequenceiq/authorization/info/model/RightV4.java

```diff
@@ -12,6 +12,12 @@
 public enum RightV4 {
     // account level
     ENV_CREATE(AuthorizationResourceAction.CREATE_ENVIRONMENT),
+    LIST_ASSIGNED_ROLES(AuthorizationResourceAction.LIST_ASSIGNED_ROLES),
```

Comment:
is this an account level right?

Reply:
@horadla23 Yes, it's assigned with IamUser